### DISCUSSION TITLE
Is there a way to create "short list" of directories?

### DISCUSSION THREAD

**Discussion by dzintars** [nvim-tree/nvim-tree.lua#1625] [Q&A]
I have a large repository with many nested directories.
At every given day i work mostly in a few quite deep sub-trees.
I would like to have something like bookmarking or marking directory (or file) to be included in some sort of "short list (or subtree)" and to open this "short list" in "tab" in the same tree window. Then i want to toggle to "short list" and back to main tree. Directories in "short list" root level can't be added, but they can be renamed or deleted. Removing from "short list" and deleting directory are two distinct operations. Everything in 2nd+ level can be added, deleted, etc.
Any ideas, how i could accomplish something like that with any other tools?
FZF is not what i want. I want to see tree view of selected subset of my repository directories/files.

**Comment by alex-courtis** [comment]
We could achieve this via `filters.custom` which contains a blacklist of the unwanted files/directories. You could then toggle via `toggle_custom`, default `U`.

That's not a "live" list however it should achieve your goal. Please try it out and report.
